src/compas_fab/backends/kinematics/utils.py

```diff
@@ -0,0 +1,24 @@
+import math
+
+
+def get_smaller_angle(angle):
```

Comment:
Am I mistaken or could this be simplified to something like 
```
angle = angle % (2 * math.pi)
if angle > math.pi:
    angle -= 2*math.pi
```
?

Reply:
👍 good catch!